build.assets/makefiles/master/k8s-master/cluster-dns.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Prior to upgrade to Kubernetes 1.19 the list of cluster DNS servers are provided
+# through the kubelet flag `--cluster-dns`. This flag is now deprecated and the cluster
+# DNS servers must be specified via the kubelet config file under `clusterDNS`.
+#
+# This script writes the cluster DNS servers into the kubelet config file.
+#
+# More info on `--cluster-dns` can be found at: https://v1-19.docs.kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
+
+# Environment file that contains the cluster DNS_ADDRESSES.
+DNS_ENV=/run/dns.env
+# Path to kubelet config file.
+KUBELET_CONFIG=/etc/kubernetes/kubelet.yaml
+
+# If /run/dns.env file does not exist exit without any changes.
```

Comment:
I believe it's possible to race during startup, so I think you want to exit with an error so that kubelet doesn't start until the file exists. 

Although I guess to be fair the file is still listed as an env file in the unit, so maybe this shouldn't really be possible unless something really funny occurs.